habitat/sims/habitat_simulator.py

```diff
@@ -138,8 +138,8 @@ def get_observation(self, sim_obs):
 
 
 @registry.register_simulator(name="Sim-v0")
-class HabitatSim(Simulator):
-    """Simulator wrapper over habitat-sim
+class HabitatSim(habitat.Simulator):
```

Comment:
```suggestion
class HabitatSim(Simulator):
```
This is why tests are failing

Reply:
thanks, this snuck in when I was resolving merge conflicts.